incubator/kiam/templates/_helpers.tpl

```diff
@@ -24,9 +24,67 @@ If release name contains chart name it will be used as a full name.
 {{- end -}}
 {{- end -}}
 
+{{/*
+Create a fully qualified agent name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "kiam.agent.fullname" -}}
+{{- if .Values.agent.fullnameOverride -}}
```

Comment:
Do you need agent and server names to be configurable? Can you just have the standard fullname template that's base on the chart name and append suffixes for agent and server, similar to what you had before?

Reply:
Can do; I just tweaked based on the Prometheus chart.